docs/ref/tasks.txt

```diff
@@ -0,0 +1,318 @@
+=====
+Tasks
+=====
+
+.. module:: django.tasks
+   :synopsis: Django's built-in background task system.
+
+Task definition
+===============
+
+.. function:: task
+
+A decorator defining a :class:`Task`::
+
+    from django.tasks import task
+
+
+    @task
+    def calculate_meaning_of_life() -> int:
+        return 42
+
+In the above example, ``calculate_meaning_of_life`` is a :class:`Task` instance.
+
+``task`` accepts a number of arguments, which are used as-is for the ``Task``:
+
+* :attr:`priority <Task.priority>`
+* :attr:`backend <Task.backend>`
+* :attr:`queue_name <Task.queue_name>`
+* :attr:`enqueue_on_commit <Task.enqueue_on_commit>`
+
+If the Task is not valid, according to the backend,
```

Comment:
```suggestion
If the ``Task`` is not valid, according to the backend,
```
I would prefer the "Task" to have code quotation (applied everywhere) or be a link to the Task class